embedded-graphics/src/primitives/arc/linear_equation.rs

```diff
@@ -0,0 +1,48 @@
+use crate::geometry::{Angle, Point, Real, Trigonometry};
+
+/// Define one side of a line
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
+pub(in crate::primitives) enum LineSide {
+    Above,
+    Below,
+}
+
+/// Linear equation representation
+///
+/// The equation is stored as the a, b and c coefficients of the ax + by + c = 0 equation
+#[derive(Copy, Clone, PartialEq, PartialOrd, Debug)]
+pub(in crate::primitives) struct LinearEquation {
+    a: Real,
+    b: Real,
+    c: Real,
+}
+
+impl LinearEquation {
+    /// Create a new linear equation based on one point and one angle
+    pub(in crate::primitives) fn from_point_angle(point: Point, angle: Angle) -> Self {
```

Comment:
Related to the comment above: If the whole struct is only `pub(in crate::primitives)`, do we also need to limit the scope of each constructor? This way we would need to change multiple locations, if we ever decide to widen the scope of, in this example, `LinearEquation`.

Reply:
Ah, I see what you mean now. That's a good point. I think I can just make these `pub` and rely on the struct visibility as you suggest. I didn't think of that. I'll change the arc/sector methods accordingly.